### DISCUSSION TITLE
docker部署archer，archer服务出现问题，无法访问

### DISCUSSION THREAD

**Discussion by GDFboy** [hhyo/Archery#1904] [General]
报错日志
2022/10/10 14:38:35 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: read udp 127.0.0.1:42601->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:35 Problem with dial: dial tcp: lookup redis on 127.0.0.11:53: read udp 127.0.0.1:55996->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:42 Problem with dial: dial tcp: lookup redis on 127.0.0.11:53: read udp 127.0.0.1:59794->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:42 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: read udp 127.0.0.1:60644->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:49 Problem with dial: dial tcp: lookup mysql on 127.0.0.11:53: read udp 127.0.0.1:47633->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:49 Problem with dial: dial tcp: lookup redis on 127.0.0.11:53: read udp 127.0.0.1:59356->127.0.0.11:53: i/o timeout. Sleeping 1s
2022/10/10 14:38:54 Timeout after 1m0s waiting on dependencies to become available: [tcp://mysql:3306 tcp://redis:6379]
2022/10/10 14:38:57 Waiting for: tcp://mysql:3306
2022/10/10 14:38:57 Waiting for: tcp://redis:6379

**Comment by hhyo** [comment]
连不上redis和mysql

**Reply by GDFboy** [reply]
但是我初始化库的时候，基础库是能够被创建出来了
